crates/usvg-parser/src/stream.rs

```diff
@@ -0,0 +1,448 @@
+// Copyright 2018 the SVG Types Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use crate::Error;
+
+/// Extension methods for XML-subset only operations.
+pub(crate) trait ByteExt {
```

Comment:
There are already functions in Rust for some of these:
Example: https://doc.rust-lang.org/std/primitive.char.html#method.is_digit